gender_analysis/gender.py

```diff
@@ -136,3 +142,66 @@ def get_pronouns(self):
                 pronouns.add(pronoun)
 
         return pronouns
+
+    def uses_name(self, name):
```

Comment:
I'd strike this method completely -- since it's just doing an `in` on an attribute of the class, the user can just as easily write that themselves. This is additional code that doesn't actually extend any functionality.

The doctest is useful and is good documentation, though, so we should maintain that and put elsewhere.